### PR TITLE
Added support for fragment arguments

### PR DESCRIPTION
**Pull request by alixwar**
Fixes https://github.com/robolectric/robolectric/issues/3296

### Overview
It's not possible to specify arguments for a Fragment using the current FragmentController API.
This was possible using the old API: 

```
Fragment fragment = new FooFragment();
Bundle arguments = new Bundle();
// Add arguments... 
fragment.setArguments(arguments);
FragmentTestUtil.startFragment();
```

### Proposed Changes
Add a method, `arguments()` to `FragmentController` so that it's possible to do:

```
Bundle arguments = new Bundle();
// Add arguments... 
Robolectric.buildFragment(FooFragment.class).arguments(arguments).start();
```

### CODE REVIEW

File: robolectric/src/main/java/org/robolectric/android/controller/FragmentController.java [hunk 1]
```diff
@@ -123,6 +123,11 @@ public void run() {
     return this;
   }
 
+  public FragmentController<F> arguments(Bundle bundle) {
```
Comment (jongerrish) [inline robolectric/src/main/java/org/robolectric/android/controller/FragmentController.java]: You might want to make this a second constructor argument, then have an overloaded buildFragment(..., Bundle) method, this way its not possible to call the builder methods out of order. We had that issue on withIntent() for Activities.

Comment (alixwar) [reply robolectric/src/main/java/org/robolectric/android/controller/FragmentController.java]: I don't see any `buildFragment()` methods to overload. Do you mean `create()`?

Comment (alwa) [reply robolectric/src/main/java/org/robolectric/android/controller/FragmentController.java]: Disregard my comment. Found what you meant. Will look into it.

Comment (alixwar) [reply robolectric/src/main/java/org/robolectric/android/controller/FragmentController.java]: @jongerrish  Updated as per your comment. Let me know if this change was what you requested

**Comment by jongerrish** [comment]
Thanks for the contribution!

**Pull request merged**